examples/microwave-oven-app/microwave-oven-common/include/microwave-oven-device.h

```diff
@@ -35,13 +35,21 @@ namespace Clusters {
 /**
  * set default value for the optional attributes
  */
-constexpr uint8_t kDefaultMinPower  = 10u;
-constexpr uint8_t kDefaultMaxPower  = 100u;
-constexpr uint8_t kDefaultPowerStep = 10u;
+constexpr uint8_t kMinPower            = 20u;
+constexpr uint8_t kMaxPower            = 110u;
+constexpr uint8_t kPowerStep           = 10u;
+constexpr uint32_t kMaxCookTime        = 86400u;
+constexpr uint8_t kDefaultPowerSetting = 100u;
 
 constexpr uint8_t ModeNormal  = 0;
 constexpr uint8_t ModeDefrost = 1;
 
+constexpr uint16_t kExampleWatt1 = 100u;
+constexpr uint16_t kExampleWatt2 = 300u;
+constexpr uint16_t kExampleWatt3 = 500u;
+constexpr uint16_t kExampleWatt4 = 800u;
+constexpr uint16_t kExampleWatt5 = 1000u;
```

Comment:
do these have to be global constants or could they be part of the example microwave class? a constant named `chip::app::Clusters::kExampleWatt1` is not as obvious of watt for what if it gets expanded outside of this header and loses context. Same for all the others. if it has to be in the header, then another namespace may make sense.